okhttp-testing-support/src/main/kotlin/okhttp3/OkHttpClientTestRule.kt

```diff
@@ -136,8 +136,13 @@ class OkHttpClientTestRule : TestRule {
   }
 
   private fun ensureAllTaskQueuesIdle() {
+    val entryTime = System.currentTimeMillis()
```

Comment:
If you’re measuring durations, you gotta use nanoTime. In this case it doesn’t really matter but I prefer to do the proper thing everywhere.